### DISCUSSION TITLE
Escape HTML

### DISCUSSION THREAD

**Discussion by developit** [community/community#8530] [Projects and Issues]
Creating a "Draft Issue" in the new Projects UI that contains something like `<3` will truncate whatever text follows the `<` because HTML is not escaped.

**Comment by TravisSpomer** [comment]
In addition, if you edit the draft issue in the full-screen UI, and type HTML there, that gets converted into HTML elements when you press Enter!

`<input type=checkbox checked> <span style="color: red;">This seems sketchy</span>`

![image](https://user-images.githubusercontent.com/16262858/162638951-51c43696-9164-4470-b077-908213034c87.png)
